src/btree/bt_split.c

```diff
@@ -861,6 +861,10 @@ __split_parent(WT_SESSION_IMPL *session, WT_REF *ref, WT_REF **ref_new,
 			}
 		}
 
+		/* Check that we are not discarding active history. */
+		WT_ASSERT(session, (next_ref == ref && discard) ||
+		    !__wt_page_las_active(session, next_ref));
```

Comment:
This is a fixed version of the previous assertion I had here.  When a page is splitting, that's because it has been just been reconciled, so it's okay if there was an active `ref->page_las` -- we must have read that data into cache before reconciling.

Otherwise, we're splitting into a parent page and discarding a sibling that appeared empty to reconciliation but had active history in lookaside.